govc/session/login.go

```diff
@@ -149,16 +147,30 @@ func (cmd *login) issueToken(ctx context.Context, vc *vim25.Client) (string, err
 	req := sts.TokenRequest{
 		Certificate: c.Certificate(),
 		Userinfo:    cmd.Userinfo(),
-		Delegatable: true,
-		ActAs:       cmd.actas,
+		Renewable:   true,
+		Token:       cmd.token,
 		Lifetime:    cmd.life,
 	}
 
+	if req.Certificate == nil {
+		req.Delegatable = true // Bearer token request
+	}
+
 	s, err := c.Issue(ctx, req)
 	if err != nil {
 		return "", err
 	}
 
+	duration := s.Lifetime.Expires.Sub(s.Lifetime.Created)
+	if duration != req.Lifetime && req.Token != "" {
```

Comment:
Do we need to consider the case where `duration > req.Lifetime`? I would hope that would not be possible, but in that case it doesn't seem like we'd want to renew.